### PR TITLE
introduced unidecode while doing encoding-decoding in sanitize

### LINKED ISSUE #37: Use Unidecode or similar when converting to ASCII
When converting to ASCII (which I would suggest doing as a function separate from `sanitize`), I would suggest using Unidecode or similar to avoid removing characters and instead use ASCII alternatives when possible.

### PR DESCRIPTION
**Pull request by sagararora19992**
Fixes https://github.com/scrapinghub/shublang/issues/37

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -18,4 +18,5 @@ pytest==5.2.1
 six==1.12.0
 w3lib==1.21.0
 wcwidth==0.1.7
-zipp==0.6.0
\ No newline at end of file
+zipp==0.6.0
+unidecode==0.4.20
```
Comment (Gallaecio) [inline requirements.txt:22]: :lipstick: What about keeping the file in alphabetical order?

File: shublang/shublang.py [hunk 2]
```diff
@@ -11,6 +11,7 @@
 from price_parser import Price
 import types
 from urllib import parse
+from unidecode import unidecode
```
Comment (Gallaecio) [inline shublang/shublang.py:14]: :lipstick: Same here, although the existing order does seem rather chaotic.

**Review by Gallaecio — COMMENTED** [review]
Looks like a good start, thanks!

You also need to add the dependency to `setup.py`, though.

It would also be great if you could update the tests to cover this change.

**Comment by VMRuiz** [comment]
Implements https://github.com/scrapinghub/shublang/issues/37

**Comment by sagararora19992** [comment]
Thank you @Gallaecio  for the review.
I have made the changes suggested by you.

**Review by Gallaecio — APPROVED** [review]
Looks good to me, thanks!

**Pull request merged**